sqlmesh/core/config/connection.py

```diff
@@ -379,16 +379,14 @@ def _connection_kwargs_keys(self) -> t.Set[str]:
     @property
     def _static_connection_kwargs(self) -> t.Dict[str, t.Any]:
         """kwargs that are for execution config only"""
-        from sqlmesh import __version__
 
-        custom_user_agent_config = {"custom_user_agent": f"SQLMesh/{__version__}"}
```

Comment:
this can't be removed, so i don't believe this is a proper fix

Reply:
See the change to make it a query parameter